code/game/machinery/autolathe.dm

```diff
@@ -78,8 +78,9 @@
 	RefreshParts()
 
 /obj/machinery/autolathe/upgraded/gamma/Initialize()
-    . = ..()
-    adjust_hacked(TRUE)
+	files = new /datum/research/autolathe/gamma(src)
+	. = ..()
+	adjust_hacked(TRUE)
```

Comment:
```suggestion
	. = ..()
	files = new /datum/research/autolathe/gamma(src)
	adjust_hacked(TRUE)
```

Keep init order sane.

Reply:
I'm trying to remember why this wasn't working... But it breaks stuff., I'll look at a more thorough workaround